be/src/exprs/utility_functions.cpp

```diff
@@ -17,27 +17,25 @@
 
 #include "exprs/utility_functions.h"
 
-#include "exprs/expr.h"
 #include "exprs/anyval_util.h"
-#include "util/debug_util.h"
+#include "exprs/expr.h"
 #include "runtime/tuple_row.h"
+#include "util/debug_util.h"
 
 namespace doris {
 
-void UtilityFunctions::init() {
-}
+void UtilityFunctions::init() {}
 
 StringVal UtilityFunctions::version(FunctionContext* ctx) {
     return AnyValUtil::from_string_temp(ctx, "5.1.0");
 }
 
-BooleanVal UtilityFunctions::sleep(
-        FunctionContext* ctx, const IntVal& milliseconds) {
+BooleanVal UtilityFunctions::sleep(FunctionContext* ctx, const IntVal& milliseconds) {
     if (milliseconds.is_null) {
         return BooleanVal::null();
     }
-    usleep(milliseconds.val * 1000 * 1000);
+    usleep(milliseconds.val * 1000 * 1000); // TODO(hw): ms * 1000 * 1000 = ns
```

Comment:
why not change this?

Reply:
I'm confused about this. This usleep() use ns as its argument. Is it a mistake?
I should change it to
`SleepFor(MonoDelta::FromMilliseconds(milliseconds.val)` ?

Reply:
this function is used to support SQL `select sleep(N)`; N is for N second. so you just change it to `sleepFor(Second)`